endpointmanager/pkg/fhirendpointquerier/populatefhirendpoints_integration_test.go

```diff
@@ -348,8 +348,10 @@ func Test_RemoveOldEndpoints(t *testing.T) {
 	err = store.DB.QueryRow(query_str).Scan(&ct)
 	th.Assert(t, err == nil, err)
 	th.Assert(t, ct == 2, "did not persist second endpoint as expected")
+
 	endptInfo := endpointmanager.FHIREndpointInfo{
-		URL: endpt2.URL,
+		URL:      endpt2.URL,
+		Metadata: &endpointmanager.FHIREndpointMetadata{},
```

Comment:
I think it makes sense here to have a non-empty FHIREndpointMetadata field so that you can check and make sure it is still in the metadata table at the end of this test